metricbeat/etc/metricbeat.template.json

```diff
@@ -24,8 +24,56 @@
         "@timestamp": {
           "type": "date"
         },
+        "mysql-status": {
```

Comment:
This is unrelated to this change and was updated because previously I forgot to run `make fields`.